apps/web/cypress/tests/notifications.spec.ts

```diff
@@ -48,4 +48,37 @@ describe('Notification Templates Screen', function () {
 
     cy.url().should('include', '/templates/create');
   });
+
+  it('when clicking on the try digest playground it should redirect to digest playground page', function () {
+    cy.initializeSession({ noTemplates: true }).as('session');
+    cy.intercept('**/notification-templates**').as('notificationTemplates');
+    cy.visit('/templates');
+    cy.wait('@notificationTemplates');
+
+    cy.getByTestId('no-workflow-templates-placeholder').should('be.visible');
+    cy.getByTestId('try-digest-playground-tile').should('not.be.disabled');
+    cy.getByTestId('try-digest-playground-tile').click();
```

Comment:
Wondering if we can chain the actions. Though I like the readability line by line.

Reply:
yes we can chain it, will do that ;)